dataprofiler/profilers/profile_builder.py

```diff
@@ -714,17 +714,27 @@ def _restore_data_labelers(self, data_labeler=None):
             data_labeler_options = self.options.data_labeler
             use_data_labeler = data_labeler_options.is_enabled
 
-        if data_labeler_options.is_enabled \
-                and data_labeler_options.data_labeler_object is None:
+        if use_data_labeler:
             try:
-                if data_labeler is None:
+                if data_labeler is None and data_labeler_options is None:
+                    data_labeler = DataLabeler(
+                        labeler_type=self._default_labeler_type,
+                        dirpath=None,
+                        load_options=None)
+                elif data_labeler is None:
```

Comment:
This if could be just one DataLabeler construction where dirpath is a variable that is None if options is None and otherwise its pulled from options

Reply:
done

Reply:
i.e.
`dirpath = None if data_labeler_options is None else data_labeler_options.data_labeler_dirpath`